lib/amorail/entity/persistance.rb

```diff
@@ -40,5 +40,21 @@ def reload
     def extract_data_add(response)
       response.fetch('add').first
     end
+
+    def extract_data_update(response)
+      case data = response.fetch('update')
+      when Array
+        data.first
+      when Hash
+        merge_errors(data)
```

Comment:
Not sure about additional checks here.
If `data` is a Hash there can be only errors scenario (we don't now another one for now).
And if it is an errors scenario `errors` shouldn't be empty.